### PR TITLE
Release PR tempate

### LINKED ISSUE #49: create Release PR template

### PR DESCRIPTION
**Pull request by kodesiba**
Added a release Pull Request Template.  Closes #49 

To use the PR template [this article](https://github.blog/2018-01-25-multiple-issue-and-pull-request-templates/) will be helpful to explain how to use PR templates that are not the default one

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/release.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+# Release Checklist
+<!--- Fill out the following Release checklist -->
+
+- [ ] Version number has been updated
+- [ ] NEWS.md has been updated
+- [ ] README.md has been updated (if applicable)
+- [ ] Vignettes have been updated (if applicable)
+- [ ] Ensure all unit tests are passing
+- [ ] All QC Checklists in this PR are complete
```
Comment (bms63) [inline .github/PULL_REQUEST_TEMPLATE/release.md:17]: Do we have a QC Checklist?  I'm intrigued?!?

File: .github/PULL_REQUEST_TEMPLATE/release.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## Milestone
+<!--- Link to the milestone for the release. ---> 
+<!--- Make sure all relevant issues/PRs are included on the linked pages. --->
+Milestone: 
+
+# Release Checklist
+<!--- Fill out the following Release checklist -->
+
+- [ ] Version number has been updated
+- [ ] NEWS.md has been updated
```
Comment (bms63) [inline .github/PULL_REQUEST_TEMPLATE/release.md:13]:
```suggestion
- [ ] NEWS.md has been updated and issues linked where appropriate
```

File: .github/PULL_REQUEST_TEMPLATE/release.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] README.md has been updated (if applicable)
+- [ ] Vignettes have been updated (if applicable)
+- [ ] Ensure all unit tests are passing
```
Comment (bms63) [inline .github/PULL_REQUEST_TEMPLATE/release.md:16]: Should we enforce a certain level of test coverage - or at least no see if fall below the previous released test coverage?

**Review by bms63 — CHANGES_REQUESTED** [review]
I am loving this.  Do we want to add in any quirks of dealing with CRAN people.  E.g. additional advice on anything specific to CRAN?

File: .github/PULL_REQUEST_TEMPLATE/release.md (hunk 1 above)
Comment (kodesiba) [reply .github/PULL_REQUEST_TEMPLATE/release.md]: We don't, this fell through my copy and paste check

**Review by bms63 — APPROVED** [review]

**Pull request merged**